ci/cloudbuild/builds/checkers.sh

```diff
@@ -131,14 +131,18 @@ time {
     xargs -P "$(nproc)" -n 50 -0 bash -c "sed_edit ${expressions[*]} \"\$0\" \"\$@\""
 }
 
-# Apply transformations to fix whitespace formatting in files not handled by
-# clang-format(1) above.  For now we simply remove trailing blanks.  Note that
-# we do not expand TABs (they currently only appear in Makefiles and Makefile
-# snippets).
+# Applies the following fixes in text files that do not say "DO NOT EDIT":
```

Comment:
Something about barbers that only shave those that do not shave themselves... j/k

Reply:
also, s/do not say/unless they say/ maybe?

Reply:
"do not shave themselves"... hilarious! I didn't realize it, but it's true here :-)

Also fixed wording.

Reply:
Used a character class in the regex to avoid the shaving barbers problem, now `\b[D]O NOT EDIT\b` won't match this file itself.